scripts/ci_utils.sh

```diff
@@ -3,6 +3,22 @@
 # All functions expect to be executed the root directory of the repository, and
 # will exit with this as the current directory.
 
+
+# Init some global variables related to the platform. Some other functions may
+# expect this to be called before they invoked.
+function init_platform() {
+    platform=`uname`
+    echo platform=${platform}
+}
+
+# Assert that init_platform has been called
+function assert_init_platform() {
+    if [ "" == "${platform}" ] ; then
```

Comment:
For consistency, in such a conditional (where a variable value is compared to a constant or an hardcoded value) the variable should be on the left, the value/constant it is compared to should be on the right: `[ "${platform}" == "" ]`

Reply:
switched value order